src/server/modules/user/index.ts

```diff
@@ -20,6 +20,10 @@ type LinkInformationProperties = {
 
 type ShortUrlOperationProperty = UserIdProperty & ShortUrlProperty
 
+type OptionalTagsProperty = {
+  tags?: string[] | undefined
```

Comment:
nit, `tags?: string[] | undefined` is equivalent to `tags?: string[]`

Reply:
done